ql/src/java/org/apache/hadoop/hive/ql/ddl/view/materialized/alter/rebuild/AlterMaterializedViewRebuildAnalyzer.java

```diff
@@ -299,7 +300,8 @@ private RelNode applyRecordIncrementalRebuildPlan(
       // First we need to check if it is valid to convert to MERGE/INSERT INTO.
       // If we succeed, we modify the plan and afterwards the AST.
       // MV should be an acid table.
-      boolean fullAcidView = AcidUtils.isFullAcidTable(mvTable.getTTable());
+      boolean fullAcidView = AcidUtils.isFullAcidTable(mvTable.getTTable())
```

Comment:
var name is confusing, maybe acidView or txnView?

Reply:
renamed to `acidView`